### PR TITLE
feat(StakingInput): show insufficient sUSD on burn to target

### PR DESCRIPTION
**Pull request by evgenyboxer**

### CODE REVIEW

File: translations/en.json [hunk 1]
```diff
@@ -228,8 +228,7 @@
 					"empty": "Enter an amount",
 					"burn": "Burn <0>sUSD</0>",
 					"max": "Max",
-					"to-target": "600%",
-					"insufficient": "Insufficient sUSD",
```
Comment (evgenyboxer) [inline translations/en.json:232]: those two are not used ^

File: sections/staking/components/StakingInput/StakingInput.tsx [hunk 2]
```diff
@@ -134,10 +137,24 @@ const StakingInput: React.FC<StakingInputProps> = ({
 					{isMint ? t('staking.actions.mint.action.empty') : t('staking.actions.burn.action.empty')}
 				</StyledCTA>
 			);
+		} else if (
+			burnType === BurnActionType.TARGET &&
+			maxBurnAmount != null &&
+			burnAmountToFixCRatio != null &&
+			burnAmountToFixCRatio.isGreaterThan(maxBurnAmount)
```
Comment (evgenyboxer) [inline sections/staking/components/StakingInput/StakingInput.tsx:144]: this is the main condition that needs to be checked ^

File: sections/staking/components/BurnTiles/BurnTiles.tsx [hunk 3]
```diff
@@ -17,6 +25,8 @@ type BurnTilesProps = {
 };
 
 const burnIcon = <Svg src={BurnCircle} />;
+const burnCustomIcon = <Img src={BurnCustomCircle} />; // TODO: investigate why it doesn't render correctly with <Svg /> (ids were replaced to be unique)
```
Comment (andytcf) [inline sections/staking/components/BurnTiles/BurnTiles.tsx:28]: @evgenyboxer could it be the same issue as you told me, the mask-type

**Review by andytcf — APPROVED** [review]
LGTM

File: sections/staking/components/BurnTiles/BurnTiles.tsx (hunk 3 above)
Comment (evgenyboxer) [reply sections/staking/components/BurnTiles/BurnTiles.tsx:28]: doesn't seem so... after more testing, it seems that it conflicts with `burn-circle.svg` but I couldn't find any matching ids...

**Pull request merged**